tst/standard/ReducibleMatrixGroups.tst

```diff
@@ -74,6 +74,63 @@ Error, <d> must be even
 gap> SpStabilizerOfNonDegenerateSubspace(4, 2, 3);
 Error, <k> must be less than <d> / 2
 
+#
+gap> TestOmegaStabilizerOfNonDegenerateSubspace := function(epsilon, d, q, epsilon_0, k)
+>   local G;
+>   G := OmegaStabilizerOfNonDegenerateSubspace(epsilon, d, q, epsilon_0, k);
+>   Assert(0, CheckSize(G));
+>   Assert(0, IsSubset(Omega(epsilon, d, q), GeneratorsOfGroup(G)));
```

Comment:
It'd be preferable to add a function `IsSubsetOmega` in `tst/utils.g`. It could for now look like this:
```
IsSubsetSU := function(epsilon, n, q, G)
  Assert(0, DimensionOfMatrixGroup(G) = n);
  Assert(0, DefaultFieldOfMatrixGroup(G) = GF(q));
  Assert(0, CheckGeneratorsSpecial(G));
  Assert(0, CheckQuadraticForm(G));
  return true;
end;
```
But of course this does not yet verify `epsilon`, nor that we are really in `Omega`.... I guess for that we'll want `SpinorNorm` from https://github.com/gap-system/gap/pull/4668 (so only in GAP 4.12). Err, and yet more code in char 2... *sigh*. Certainly nothing to delay this PR over, but we should track this in an issue.